static/js/matcher.js

```diff
@@ -14,3 +14,13 @@ function changeThis(sender) {
         document.getElementById("changed-name-2").removeAttribute('disabled');
     }
 };
+
+// Save data to sessionStorage
+function createItem(sender) {
```

Comment:
I don't think that the function parameter is used anywhere, so not much point in having it?